cibyl/models/ci/system.py

```diff
@@ -277,3 +277,23 @@ def add_toplevel_model(self, model):
         else:
             # Add a brand-new tenant
             self.tenants[key] = model
+
+
+class ZuulDSystem(JobsSystem):
```

Comment:
I don't think an additional system is necessary. Given that it's essentially JobsSystem with a default value for system type I think we can keep working with that without introducing another one. Also, wouldn't ZuulSystem be more appropiate for this source?

Reply:
Thanks, I removed `ZuulDSystem`. 

`ZuulSystem` configured with different output formats  for ex:
```
      Tenant: packstack-centos9-on-rdo
        Total jobs found in query: 0
      Tenant: packstack-multinode-centos9-on-rdo
        Total jobs found in query: 0
      Tenant: packstack-centos9-integration-scenario001
        Total jobs found in query: 0
      Tenant: packstack-centos9-integration-scenario002
        Total jobs found in query: 0
      Tenant: packstack-centos9-integration-scenario003
        Total jobs found in query: 0
      Tenant: packstack-centos9-multinode-scenario002-tempest
        Total jobs found in query: 0
      Total tenants found in query: 1901
```

To get jobs listed `JobsSystem` is good fit.
```
     Job: packstack-centos9-on-rdo
      Job: packstack-multinode-centos9-on-rdo
      Job: packstack-centos9-integration-scenario001
      Job: packstack-centos9-integration-scenario002
      Job: packstack-centos9-integration-scenario003
      Job: packstack-centos9-multinode-scenario002-tempest
      Total jobs found in query: 1901
```